src/client/components/GatewayErrorSummary.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+	ErrorSummary,
+	ErrorSummaryProps,
+} from '@guardian/source-development-kitchen/react-components';
+
+import { asStructuredError, GatewayError } from '@/shared/model/Errors';
+import {
+	errorContextLastTypeSpacing,
+	errorContextSpacing,
+	errorMessageStyles,
+} from '@/client/styles/Shared';
+
+export interface GateWayErrorSummaryProps
+	extends Omit<ErrorSummaryProps, 'message' | 'cssOverrides'> {
+	shortRequestId?: string;
+	gatewayError?: GatewayError;
+}
+export default function GatewayErrorSummary(props: GateWayErrorSummaryProps) {
```

Comment:
In Gateway we generally don't use `default` exports, preferring named exports

```suggestion
export function GatewayErrorSummary(props: GateWayErrorSummaryProps) {
```

or to be even more "Gateway"-like

```suggestion
export const GatewayErrorSummary = (props: GateWayErrorSummaryProps) => {
```